experimental/handoff-library/csharp_dotnetcore/samples/LivePersonAgentBot/PostDeployScripts/IncludeSources.targets

```diff
@@ -0,0 +1,15 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
It's been a while since I looked at an ASP.Net application that did this. Is this still necessary or can this file be removed? I don't think it's needed.